openui5/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": false,
```

Comment:
Prefer to make at least `noImplicitAny` true. We normally prefer `strictNullChecks` too, but if the documentation you're parsing doesn't have nullability information, that's OK.